src/simulation/dynamics/linearTranslationalBodies/linearTranslationBodiesOneDOF/linearTranslationOneDOFStateEffector.h

```diff
@@ -33,7 +33,7 @@
 #include "architecture/messaging/messaging.h"
 
 /*! @brief linear spring mass damper state effector class */
-class linearTranslationOneDOFStateEffector :
```

Comment:
This is going to break existing code.  I agree with the fix, but can we deprecate this class name fix?  Could we use

```
class MyClass {
    // ...
};

using OtherName = MyClass;

OtherName x;   // same as MyClass
```
to achieve this and keep backwards compatibility for a year?  

Next question, can we give a deprecation warning if someone uses the old name?